EverestSplash/EverestSplash.cs

```diff
@@ -134,28 +138,39 @@ public class EverestSplashWindow {
 
     private readonly NamedPipeClientStream ClientPipe;
     private WindowInfo windowInfo;
-    private readonly FNAFixes fnaFixes = new();
     private readonly string targetRenderer;
     private readonly Assembly currentAssembly;
 
     private FontLoader renogareFont = null!;
     private LoadingProgress _loadingProgress = new(0, 0, "");
+    private readonly Randomness randomness;
+
+    private bool rightSidedWheel = false;
+    public string LoadingText = "Loading %s";
 
     private LoadingProgress loadingProgress {
         get => _loadingProgress;
         set {
             if (renogareFont == null) return; // Too early :/ (ignore data received when the splash was still initializing)
             _loadingProgress = value with { lastMod = value.lastMod == "" ? _loadingProgress.lastMod : value.lastMod };
+            if (_loadingProgress.raw) { // .raw means no extra decorations on the mod name
+                // also skip sanitization since this is not arbitrary data
+                windowInfo.modLoadingProgressCache?.SetText(_loadingProgress.lastMod);
+                return;
+            }
             char[] sanitizedName = _loadingProgress.lastMod.ToCharArray();
             // Sanitize the sent mod name, it could contain forbidden characters
-            // I KNOW I KNOW, this is absolutely slow and painful to your eyes, TWO whole string copies and a loop O(n*3), painful,
+            // I KNOW I KNOW, this is absolutely slow and painful to your eyes, TWO whole string copies, and a loop, O(n*3), painful
             // so feel free to optimize it :D
             for (int i = 0; i < sanitizedName.Length; i++) {
                 if (!renogareFont.IsValidChar(sanitizedName[i]))
                     sanitizedName[i] = '?'; // Fallback char
             }
-
-            windowInfo.modLoadingProgressCache?.SetText("Loading " + new string(sanitizedName) + " [" + _loadingProgress.loadedMods + "/" + _loadingProgress.totalMods + "]");
+            
+            
+            // windowInfo.modLoadingProgressCache?.SetText($"Loading {sanitizedName.AsSpan()} [{_loadingProgress.loadedMods}/{_loadingProgress.totalMods}");
+            windowInfo.modLoadingProgressCache?.SetText(
+                $"{LoadingText.Replace("%s", new string(sanitizedName))} [{_loadingProgress.loadedMods}/{_loadingProgress.totalMods}]");
```

Comment:
Use `{0}` as placeholder with `string.Format()` instead of handbuilding formatting syntax using %.